Go.gitignore

```diff
@@ -13,3 +13,6 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+
+# Delve debugger compiled binary
+/__debug_bin
```

Comment:
```suggestion
__debug_bin*
```

There's a new change to delve to include timestamp in output
source: https://github.com/go-delve/delve/commit/801a9109c7fd7be8b060b466cc041a3695455c77